cmd/edit/cluster/cmd.go

```diff
@@ -1142,3 +1189,163 @@ func confirmMigration() (bool, error) {
 		Default: false,
 	})
 }
+
+// AutoNode helper functions
+func autonodeExists(cluster *cmv1.Cluster) bool {
+	return cluster.AutoNode() != nil && cluster.AutoNode().Mode() == "enabled"
+}
+
+func autonodeRoleExists(cluster *cmv1.Cluster) string {
+	if cluster.AWS() != nil && cluster.AWS().AutoNode() != nil {
+		return cluster.AWS().AutoNode().RoleArn()
+	}
+	return ""
+}
+
+func setAutoNode(r *rosa.Runtime, cmd *cobra.Command, cluster *cmv1.Cluster, autonode string, roleArn string) (
+	mode string, role string, err error) {
```

Comment:
Can we change the name of `mode string` to `autonodeMode string`? it's a bit confusing later on in the function

Reply:
fixed on last commit